### PR TITLE
Use boostrap autoapprover service account directly

### PR DESCRIPTION
**Pull request by kargakis**
Fixes https://github.com/openshift/openshift-azure/issues/158

/hold

### CODE REVIEW

**Comment by kargakis** [comment]
@openshift/sig-azure any preferences over keeping the bootstrap-autoapprover in the `openshift-azure` vs `openshift-infra` namespace?

**Comment by kwoodson** [comment]
Infra makes more sense to me.

/lgtm

**Comment by pweil-** [comment]
infra is fine for me too

File: pkg/addons/data/SecurityContextConstraints.security.openshift.io/privileged.yaml [hunk 1]
```diff
@@ -36,7 +36,6 @@ supplementalGroups:
 users:
 - system:admin
 - system:serviceaccount:openshift-infra:build-controller
-- system:serviceaccount:openshift-azure:default
```
Comment (kargakis) [inline pkg/addons/data/SecurityContextConstraints.security.openshift.io/privileged.yaml]: @jim-minter unclear why this was needed.

**Comment by kargakis** [comment]
Needs one more review 

/cc @mjudeikis @kwoodson

File: pkg/addons/data/SecurityContextConstraints.security.openshift.io/privileged.yaml (hunk 1 above)
Comment (mjudeikis) [reply pkg/addons/data/SecurityContextConstraints.security.openshift.io/privileged.yaml]: I think it is leftover from tunnel

Comment (kargakis) [reply pkg/addons/data/SecurityContextConstraints.security.openshift.io/privileged.yaml]: Ah, right, cool.

**Pull request merged**